test/evaluation.spec.js

```diff
@@ -226,8 +226,11 @@ module.exports.addTests = function({testRunner, expect}) {
       expect(error.message).toContain('JSHandles can be evaluated only in the context they were created');
     });
     it('should simulate a user gesture', async({page, server}) => {
-      const result = await page.evaluate(() => document.execCommand('copy'));
-      expect(result).toBe(true);
+      await page.evaluate(() => document.body.appendChild(document.createTextNode('test')));
+      const selectResult = await page.evaluate(() => document.execCommand('selectAll'));
+      expect(selectResult).toBe(true);
```

Comment:
Firefox is unhappy about this. Yay! Browsers!

Let's just do this with window.getSelection to not mess around more with execCommand than we have to, or just don't assert that the selectAll worked.

Reply:
Updated expectations to check only result of 'copy'.